samples/cellular/modem_shell/src/at/at_shell.c

```diff
@@ -97,6 +99,19 @@ static int cmd_at(const struct shell *shell, size_t argc, char **argv)
 		shell_help(shell);
 	} else {
 		k_mutex_lock(&mosh_at_resp_buf_mutex, K_FOREVER);
+
+		/* For the AT%CMNG command, convert '\\n' to '\r\n'
+		 * to allow writing of multi-line certificates.
```

Comment:
I think it would be good to mention this tweak in the documentation for `at` command. There are few other notes already so adding one for this would be good.